doc/_static/style.css

```diff
@@ -203,3 +203,19 @@ a.copybtn {
 .sphx-glr-single-img {
     max-width: 80%!important;
 }
+
+/*
```

Comment:
Here is the CSS styles from myst-nb.
```
div.cell_output tr,
div.cell_output th,
div.cell_output td {
  text-align: right;
  vertical-align: middle;
  padding: 0.5em 0.5em;
  line-height: normal;
  white-space: normal;
  max-width: none;
  border: none;
}
```
The `text-align` is set to `right`, which makes sense for displaying objects like `pandas.DataFrame` (an example at https://myst-nb.readthedocs.io/en/latest/render/format_code_cells.html#remove-stdout-or-stderr).

In MyST, the table cell alignment can be controlled using `:` characters (https://myst-parser.readthedocs.io/en/latest/syntax/tables.html), which is done by assigning `text-left`/`text-center`/`text-right` to cells. These CSS classes are not defined in Sphinx_rtd_theme, so we have to define them here.